### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by dopplershift**
- Add Jackson St. page
- Add bullets for other known workshops

### CODE REVIEW

File: events/jackson2018.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+| 10:00 | Break                                        |             |  0:15  |
+| 10:15 | [Working with Satellite Data](https://github.com/Unidata/unidata-python-workshop/blob/master/notebooks/Satellite_Data/Working%20with%20Satellite%20Data.ipynb)                  | John        |  1:00  |
+| 11:15 | Break                                        |             |  0:15  |
+| 11:30 | [Surface Data Analysis](https://github.com/Unidata/unidata-python-workshop/blob/master/notebooks/Surface_Data/Surface%20Data%20with%20Siphon%20and%20MetPy.ipynb)                        | Ryan        |  1:00  |
+| 12:30 | Lunch                                        |             |  1:15  |  
+| 13:45 | [Model Data](https://github.com/Unidata/unidata-python-workshop/blob/master/notebooks/Model_Output/Downloading%20model%20fields%20with%20NCSS.ipynb)                                   | Ryan        |  1:00  |
+| 14:45 | Break                                        |             |  0:15  |
+| 15:00 | [Time Series Analysis of Buoy Data](https://github.com/Unidata/unidata-python-workshop/blob/master/notebooks/Time_Series/Basic%20Time%20Series%20Plotting.ipynb)            | Hailey      |  1:00  |
```
Comment (jrleeman) [inline events/jackson2018.md]: Remove Hailey - I'll take this one

File: events/jackson2018.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+| 16:00 | End of day 2                                 |             |        |
```
Comment (jrleeman) [inline events/jackson2018.md]: Our flight is at 6:17 - should we plan on ending early here?

File: events/jackson2018.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+| 10:40 | Break                                        |             |  0:20  |
+| 11:00 | [Introduction to NumPy](https://github.com/Unidata/unidata-python-workshop/blob/master/notebooks/NumPy/Numpy%20Basics.ipynb)                        | Ryan        |  1:00  |
+| 12:00 | [Introduction to MetPy](https://github.com/Unidata/unidata-python-workshop/blob/master/notebooks/Metpy_Introduction/Introduction%20to%20MetPy.ipynb)                        | John        |  0:30  |
+| 12:30 | Lunch                                        |             |  1:15  |
+| 13:45 | [Introduction Siphon](https://github.com/Unidata/unidata-python-workshop/blob/master/notebooks/Siphon/Siphon%20Overview.ipynb)                          | Ryan        |  1:00  |
+| 14:45 | Break                                        |             |  0:15  |
+| 15:00 | [Upper Air Data Analysis and the Skew-T](https://github.com/Unidata/unidata-python-workshop/tree/master/notebooks/Skew_T)       | Jon         |  1:00  |
```
Comment (jrleeman) [inline events/jackson2018.md]: Change to me

File: index.md [hunk 4]
```diff
@@ -21,7 +21,9 @@ You can also just view the notebooks on
 to get a feel for what's in the workshop.
 
 ## Upcoming Events
-There are no upcoming workshops.
+- [Jackson St. University](events/jackson2018.html) (30-31 August 2018 Jackson, MS)
+- Fall 2018 Unidata Workhop (22-24 October 2018 Boulder, CO)
```
Comment (jrleeman) [inline index.md]: Maybe put SJSU with dates TBD?

File: events/jackson2018.md (hunk 2 above)
Comment (dopplershift) [reply events/jackson2018.md]: Rough sketch, we can adjust later...eh, I'll bump to 3:45.

File: events/jackson2018.md (hunk 1 above)
Comment (dopplershift) [reply events/jackson2018.md]: ok

File: events/jackson2018.md (hunk 3 above)
Comment (dopplershift) [reply events/jackson2018.md]: ok

File: index.md (hunk 4 above)
Comment (dopplershift) [reply index.md]: ok

**Pull request merged**